src/main/java/gov/nasa/pds/harvest/dao/MetadataWriter.java

```diff
@@ -19,7 +19,8 @@
 
 public class MetadataWriter implements Closeable
 {
-    private final static String WARN_SKIP = "Skipping registered product ";
+    private final static String WARN_SKIP_PRE = "Skipping registered product ";
+    private final static String WARN_SKIP_POST = " (LIDVID/LID is is already registered in Elasticsearch)";
```

Comment:
I thought we switched to OpenSearch 🤷

Reply:
Yes we did, let's updat the message to "... already registered in the registry".

Reply:
Fixed